docs/content/config/security/fail2ban.md

```diff
@@ -56,6 +56,12 @@ You can manage F2B with the `setup` script. The usage looks like this:
 docker exec <CONTAINER NAME> setup fail2ban [<ban|unban> <IP>]
 ```
 
+### View log file
+
+```bash
+docker exec <CONTAINER NAME> setup fail2ban log
+```
+
 When just running `setup fail2ban`, the script will show all banned IP addresses.
```

Comment:
Because we now have another three-level heading, I'd like to either move this sentence to the top or provide a dedicated heading for how to just list all banned IPs.

Reply:
Sorry, not sure what you exactly mean. But feel free to directly change it.

Reply:
Done in [ecf2b55](https://github.com/docker-mailserver/docker-mailserver/pull/3299/commits/ecf2b55117eb88a0335299dee92a542d600a8f91)